src/scandipwa/app/plugin/TransactionId.plugin.js

```diff
@@ -0,0 +1,41 @@
+export const TRANSACTION_DATA = {
+    transaction_id: 0,
+    mutation_name: 's_placeOrder'
+};
+
+const after_getOrderField = (args, callback, instance) => {
+    return callback(...args).addFieldList(['transaction_id']);
+}
+
+const around_fetchMutation = (args, callback, instance) => {
+    const {
+        rawMutation: name = ''
+    } = args;
+
+    const response = callback(...args);
+
+    if (name === TRANSACTION_DATA.mutation_name) {
+        return response.then(
+            (result) => {
+                const { placeOrder: { order: { transaction_id } } } = result;
+                TRANSACTION_DATA.transaction_id = transaction_id;
+                return result;
+            }
+        )
+    }
+
+    return response;
+}
```

Comment:
1. Capitalize the `purchase.plugin.js` file name to be `Purchase.plugin.js`/
2. Create a class in the `Purchase.plugin.js` with two bound methods - one with the current that file's logic and one with this file's `around_fetchMutation`.
3. Instead of storing stuff in the `TRANSACTION_DATA`, store it in some class' property. 

Unfortunately, there is no way to write this logic otherwise at the moment, due to the source code's not-enough granularity in that specific place.
Alternatively, you can refactor the source code in that specific place, where it is hard to plug into, so that other people live happily, omitting the issue you have encountered.